appinventor/blocklyeditor/src/replmgr.js

```diff
@@ -1462,6 +1462,7 @@ Blockly.ReplMgr.getFromRendezvous = function() {
                 rs.versionurl = 'http://' + json.ipaddr + ':8001/_getversion';
                 rs.baseurl = 'http://' + json.ipaddr + ':8001/';
                 rs.android = (json.os || 'Android').toLowerCase() !== 'ios';
+                rs.hasfetchassets = rs.android
```

Comment:
Not that it matters technically, but there is a missing semicolon at the end of the line here.

Reply:
Fixed, good catch, thanks.